ipn/ipnlocal/network-lock.go

```diff
@@ -424,6 +431,9 @@ func (b *LocalBackend) NetworkLockInit(keys []tka.Key, disablementValues [][]byt
 		//    - DisablementSecret: value needed to disable.
 		//    - DisablementValue: the KDF of the disablement secret, a public value.
 		DisablementSecrets: disablementValues,
+
+		StateID1: binary.LittleEndian.Uint64(entropy[:8]),
```

Comment:
Would it be simpler to do two 8 byte reads instead of one 16 byte read that gets sliced?

Reply:
Maybe—but reading from the system entropy pool is potentially expensive, plus you'd then have to check for errors twice.